index.bs

```diff
@@ -4219,6 +4219,8 @@ manner suiting the user.</p>
    <li>
     <p>Apply the terms of the CSS specifications to |regionNode| within the following constraints,
     thus obtaining a CSS box |box| positioned relative to an initial containing block:</p>
+    <p class="note">The CSS specification application may make changes to a number of variables,
+    including |lineHeight|.</p>
```

Comment:
@silviapfeiffer am I right in thinking that those changes do _not_ then modify the height of a region by changing the value of _lines_ ? If so, I really think this needs to be stated, otherwise people might assume that the region has enough size to fit some number of lines but then after applying CSS to change lineHeight, find that it does not.

Reply:
We've been there, see https://github.com/w3c/webvtt/issues/371#issuecomment-346788337 , which is why we added this sentence. The CSS specification can change all those things including the height of the region.

Reply:
Yes we have! Without wanting to go round in circles, as noted at https://github.com/w3c/webvtt/issues/371#issuecomment-351439545 adjusting the CSS line height does not affect the value of _lines_ because there seems to be no step that goes back to modify it. If there actually is a step, then it is non-obvious and it would be helpful to point to it in this note, but if there is indeed no such step (and we don't add one), then that also would be worth stating in this note.

Reply:
`lines` does not get changed, but the height of the region might because the `lineHeight` does. What would you like me to add? I'm very confused!

Reply:
There's no mechanism for the height of the region to change based on `lineHeight` changing though, unless I've missed it? §7.1 Processing Model, bullet 12 step 1, bullet 2, sets `lineHeight` and `regionHeight` and derives the one time value `lines`. Then two bullets later the top is calculated before any CSS settings have been applied. Effectively at this point there is a fixed height. 

Then in bullet 12 step 2, there's the note introduced here that says that _lineHeight_ might be modified by CSS specification application. How is the region height then recalculated? 

Side-note: I couldn't find any spec text that maps the CSS `line-height` value to _lineHeight_. 

I'm reading the text "Let x be y" in an ES6-ish way, i.e. that the value of x is bound immediately and not re-evaluated whenever it is used.

So, IF my reasoning here is correct, then I'm asking for a sentence to be added to say that even if _lineHeight_ is changed due to CSS, the height of the region is not recalculated. OR, if my reasoning is incorrect (and please do explain), then add something like "which modifies the height of the region via XXX" where XXX is a pointer to a spec section or algorithm.